package.json

```diff
@@ -4,9 +4,17 @@
   "main": "dist/index.js",
   "umd:main": "dist/index.umd.js",
   "module": "dist/index.mjs",
-  "source": "hypertag.js",
+  "browser": "dist/index.mjs",
+  "types": "dist/hypertag.d.ts",
   "scripts": {
-    "build": "rimraf dist && microbundle",
+    "build": "rimraf dist && npm run build:cjs && npm run build:es && npm run build:umd && npm run minify",
+    "build:cjs": "tsc -m commonjs -t es2015 && mv dist/hypertag.js dist/index.js",
```

Comment:
Note: the CommonJS and UMD builds both were set to `es2015` by microbundle. I added that here to keep behavior consistent but IMO probably isn’t necessary unless old versions of Node should be supported